src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSUtils.java

```diff
@@ -57,24 +51,6 @@ public static void logItemizedWarning(final Logger logger, final Collection<Stri
     }
 
     /**
-     * Writes two objects using Kryo to specified local file path.
```

Comment:
Seven lines above this point (old code, so I can't comment there), you're doing a string concatenation in a loop:
```str += acc + ", "```
You should use a StringBuilder for situations like this.